packages/deployer/internal/storage-ast-validator.js

```diff
@@ -10,8 +12,10 @@ const { onlyRepeated } = require('@synthetixio/core-js/utils/misc/array-filters'
 
 class ModuleStorageASTValidator {
   constructor(asts, previousAsts) {
-    this.asts = asts;
-    this.previousAsts = previousAsts;
+    this.contractNodes = Object.values(asts).map(findContractDefinitions).flat();
```

Comment:
Converting the object of AST files, to an array of Contract Definition AST Nodes